Sources/WeatherSDK/Paginators.swift

```diff
@@ -0,0 +1,5 @@
+// Code generated by smithy-swift-codegen. DO NOT EDIT!
```

Comment:
Could we maybe codegen a comment for these empty files that explains why they're empty & that they're intentionally empty?